bundles/cis-k8s-1.5.1/Kptfile

```diff
@@ -1,14 +1,11 @@
-apiVersion: kpt.dev/v1alpha1
+apiVersion: kpt.dev/v1
 kind: Kptfile
 metadata:
   name: cis-k8s-v1.5.1
-packageMetadata:
-  shortDescription: CIS Benchmark Kubernetes v1.5.1
-openAPI:
-  definitions:
-    io.k8s.cli.setters.enforcementAction:
-      x-k8s-cli:
-        setter:
-          name: enforcementAction
-          value: dryrun
-          isSet: true
+info:
+  description: CIS Benchmark Kubernetes v1.5.1
+pipeline:
+  mutators:
+  - image: gcr.io/kpt-fn/apply-setters:v0.1
+    configPath: setters.yaml
```

Comment:
We should inline this `configMap` since we *don't* want it applied to the cluster.

```suggestion
    configMap:
      enforcementAction: dryrun
```

Reply:
oh this is cool, does this mean we can delete the setters.yaml file?